python/lsst/dax/imgserv/imageREST_v1.py

```diff
@@ -186,38 +187,15 @@ def _get_ds(image_type):
         return W13DeepCoaddDb
 
 
-def _file_response(image, file_name):
-    tmp_path = tempfile.mkdtemp()
-    file_path = os.path.join(tmp_path, file_name)
-    log.debug("temporary file_path=%s", file_path)
-    image.writeFits(file_path)
-    resp = _make_file_response(file_path)
-    os.remove(file_path)
-    os.removedirs(tmp_path)
-    return resp
-
-
-def _make_file_response(file_name):
-    # It would be nice to just write to 'data' instead of making a file.
-    # writeFits defined in afw/python/lsst/afw/math/background.py
-    # Using a cache of files might be desirable. We would need consistent and
-    # unique identifiers for the files.
-    try:
-        with open(file_name, 'rb') as f:
-            data = f.read()
-            f.close()
-            response = make_response(data)
-            response.headers["Content-Disposition"] = "attachment;\
-                    filename=image.fits"
-            response.headers["Content-Type"] = "image/fits"
-        return response
-    except IOError as e:
-        return _error(IOError.__name__, e.message, INTERNAL_SERVER_ERROR)
-
-
-def _error(exception, message, status_code):
-    response = jsonify({"exception": exception, "message": message})
-    response.status_code = status_code
+def _data_response(image):
+    # write image data to FITS file via data buffer
```

Comment:
I think PEP8 wants one line doc strings like this """Write image data to FITS file via data buffer."""

Reply:
I think we can skip a doc string here, as _data_response() is intended for internal use only.

Reply:
Please turn the comment into a docstring. If the comment is superfluous then that's a separate debate but I don't see a downside in making this a docstring.